src/Params/Controller.cpp

```diff
@@ -112,6 +114,7 @@ void Controller::defaults()
     NRPN.receive = 1;
 
     portamento.portamento = 0;
+    portamento.automode = 0;
```

Comment:
Very minor note, so I wouldn't worry about fixing this on this PR, but the rest of the initializers appear to be vertically aligned.

Reply:
It's easy enough to fix though, although honestly it looks like the whole section could do with a cleanup, with some initializers being aligned and some not. It's no problem for me to do an updated PR.

Reply:
I cleaned up the defaults() initializations as a separate commit. The rest of the commits are the same.